scripts/run_lighthouse_tests_test.py

```diff
@@ -158,6 +162,126 @@ def mock_popen(*unused_args: str, **unused_kwargs: str) -> MockTask:  # pylint:
             'Puppeteer script failed. More details can be found above.',
             self.print_arr)
 
+    def test_puppeteer_script_successfully_recording_failed(self) -> None:
```

Comment:
Removed this test and `test_puppeteer_script_failed_recording_failed` in new commit, since I was using them to ensure full coverage for `run_lighthouse_tests.py` where the code would check if a file was created and print different statements based on whether the file existed (comment 3 from the top).